src/main/resources/application.yaml

```diff
@@ -43,6 +43,10 @@ faf:
     secret: ${REPLAY_SECRET:banana}
     access-uri: ${REPLAY_URL:ws://localhost:8004}
     access-param: ${REPLAY_PARAM:verify}
+  chat:
+    secret: ${REPLAY_SECRET:banana}
+    access-uri: ${REPLAY_URL:ws://localhost:8005}
+    access-param: ${REPLAY_PARAM:verify}
   irc:
     secret: ${IRC_SECRET:banana}
     token-ttl: ${IRC_TOKEN_TTL:300}
```

Comment:
The env variable names must be changed. But I think we need a better name than chat secret. What is actually the difference to irc secret?